pre-pages/pgql-2.0-spec.md

```diff
@@ -1395,15 +1395,15 @@ edge labels:
 
 ```sql
 --PGQL
-SELECT ID(n), n.*
+SELECT n.*
 FROM MATCH (n:Person) ON financial_transactions
 ORDER BY "name"
 --SQL
 SELECT  *  
 FROM GRAPH_TABLE (financial_transactions 
   MATCH(n IS Person) 
-  COLUMNS(VERTEX_ID(n) as n_id, n.*)) 
-ORDER BY  "name"
+  COLUMNS(n.*)) 
+ORDER BY "name"
```

Comment:
just query outputs has to be updated for this query as well as the one above:

```
+--------------------+
| label(n) | name    |
+--------------------+
| Person   | Camille |
| Person   | Liam    |
| Person   | Nikita  |
+--------------------+
```
==>
```
+---------+
| name    |
+---------+
| Camille |
| Liam    |
| Nikita  |
+---------+
```

and above:

```
+-----------------------------+
| label(n) | number | name    |
+-----------------------------+
| Account  | 1001   | <null>  |
| Account  | 2090   | <null>  |
| Account  | 8021   | <null>  |
| Account  | 10039  | <null>  |
| Person   | <null> | Camille |
| Person   | <null> | Liam    |
| Person   | <null> | Nikita  |
| Company  | <null> | Oracle  |
+-----------------------------+
```
==>
```
+------------------+
| number | name    |
+------------------+
| 1001   | <null>  |
| 2090   | <null>  |
| 8021   | <null>  |
| 10039  | <null>  |
| <null> | Camille |
| <null> | Liam    |
| <null> | Nikita  |
| <null> | Oracle  |
+------------------+
```